packages/labs/ssr-react/src/enable-lit-ssr.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/**
+ * @fileoverview Side-effect import meant to be loaded in the **browser** before
+ * any user code is loaded. Installs hydration support for `LitElement`.
+ */
```

Comment:
why do we need this file in addition to `lit/experimental-hydrate-support.js`?

Reply:
With projects like Next.js that use the same source code for both server and client, I figured it's a convenience for users to just import this once at the top, and it'll handle the monkey patching on the server, and hydration import on the client.